vignettes/howto-dm-rows.Rmd

```diff
@@ -48,26 +48,26 @@ With the changes confirmed, you execute the method again, this time with the arg
 Note that `in_place = FALSE` is the default: you must opt in to actually change data on the database.
 
 Each method has its own requirements in order to maintain database consistency.
-These involve constraints on primary key values as they are how rows are identified.
+These involve constraints on primary key values that uniquely identify rows.
 
 | Method | Requirements |
 |--------|--------------|
```

Comment:
Are these constraints documented in the manual pages too? Could this table (not in this PR, later) be generated automatically to prevent its getting out of date?

Reply:
Good thinking.
We could use rmd-fragments to generate both .Rd and vignette docs, so that both of them stay in sync.

Reply:
Should we open an issue about thinking how to better keep vignettes and man pages in sync?

Reply:
Done! #1143